styles/pages/Home.module.scss

```diff
@@ -419,37 +433,44 @@
     flex-direction: column;
     padding: 48px 16px 48px 16px;
     border-radius: 8px;
+    align-items: center;
```

Comment:
I don't think we need lines 436, 437 and 440. Dosen't affect the styling hmm

Reply:
Resolved in commit 607f5a99ec73425562b5fd8c58e46dd6586477d1